### ISSUE TITLE
Light Levels & Passive Mob Spawning minecraft 1.20.4

### ISSUE THREAD

**Issue by Shazer749** [PyjamaMole/Skyblock#2]
I really love this mod and it is exactly what I am looking for. However, here are a few issues I have found and was wondering if there was anything I or you could do to fix them.

I downloaded the non-entity version of this mod along with the fabric API and this mod: https://www.curseforge.com/minecraft/mc-mods/logical-zoom. Not sure if either of these two mods is creating the following issues:

I tried using it in 1.20.4 to create a blank skyblock world to set up my own skyblock. In creative, I spawned a 100 by 100 platform of grass at y level 66, y level 0, and y level 30. At y level 66, nothing spawned, no passive mobs nothing.

At y level 0, a bunch of hostile mobs spawned. It was day time.

At y level 30, a bunch of hostile mobs spawned in with their light levels all screwed up.

Any help would be greatly appreciated! This mod is exactly what I am looking for, thanks!

**Comment by PyjamaMole** [comment]
I will try to see if I can replace the issue, I'll get back to you when I know more!

**Comment by PyjamaMole** [comment]
Okay after some testing, passive mobs do still spawn as normal on the first platform you mentioned. Passive mob spawning is just very very slow, and the higher up in the world you are the worse it gets. I ran the game at a higher tick rate and after some time did end up getting some sheep to spawn.

As for your second issue, for me the light levels seem to be correct, it shows a skylight level of 15 for me, however hostile mobs are spawning and also not burning up. A quick way I found to fix this is using the fill command to create a giant platform at world height and remove it again afterwards, this makes me believe the issue is related to the heightmaps. I will look into whats causing that to happen ^^
